cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -907,6 +907,66 @@ ALTER DATABASE ... SET TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]
 a|
 New sub-clause for `ALTER DATABASE`, which allows modifying the number of servers hosting a database in cluster environments.
 
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+ENABLE SERVER ...
+----
+a|
+New Cypher command for enabling servers.
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+ALTER SERVER SET OPTIONS
+----
+a|
+New Cypher command for setting options for a server.
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+RENAME SERVER ... TO ...
+----
+a|
+New Cypher command for changing the name of a server.
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+REALLOCATE DATABASES
+----
+a|
+New Cypher command for re-balancing what servers host which databases.
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+DEALLOCATE DATABASE[S] FROM SERVER[S] ...
+----
+a|
+New Cypher command for moving all databases from servers.
```

Comment:
not sure I agree, since removing doesn't imply that a different server will start hosting it

Reply:
true, still not sure I like the original sentence but not sure how to improve it either

Reply:
I'd vote for _moving_ as it does imply that the databases have a destination. The command itself _DEALLOCATE_ sort of implies that as well. I suppose it could be fleshed out a bit, but for brevity's sake, I'd say keep it as it is.